watertap/property_models/seawater_prop_pack.py

```diff
@@ -1447,6 +1449,26 @@ def rule_dh_vap_mass(
 
         self.eq_dh_vap_mass = Constraint(rule=rule_dh_vap_mass)
 
+    def _bpe_phase(self):
+        self.bpe_phase = Var(
+            self.params.phase_list,
+            initialize=5e-1,
+            bounds=(0, 1e9),
+            units=pyunits.K,
```

Comment:
As before regarding units of measurement,

Reply:
Reserved for future commit.